### PR TITLE
Reanimated animations.

### PR DESCRIPTION
**Pull request by andreyvEze**
https://user-images.githubusercontent.com/22767467/113897194-52530c00-97d3-11eb-8663-814f786bc752.mp4

### CODE REVIEW

File: src/modules/UI/components/Slider/Slider.js [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+  // Reset logic:
+  reset?: boolean,
+
+  // Disabled logic:
+  disabledText?: string,
+  disabled: boolean
+}
+
+const SLIDER_WIDTH = PLATFORM.deviceWidth >= 720 ? 680 : PLATFORM.deviceWidth - 45
```
Comment (andreyvEze) [inline src/modules/UI/components/Slider/Slider.js:26]: move these const values to the theme files

File: src/modules/UI/components/Slider/Slider.js [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+      ctx.offsetX = translateX.value
+    },
+    onActive: (event, ctx) => {
+      isSliding.value = true
+      translateX.value = clamp(event.translationX + ctx.offsetX, 0, UPPER_BOUND)
+    },
+    onEnd: () => {
+      isSliding.value = false
+
+      if (translateX.value < 3) {
```
Comment (andreyvEze) [inline src/modules/UI/components/Slider/Slider.js:56]: create const for `3`

File: src/modules/UI/components/Slider/Slider.js [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+  activityIndicator: {
+    backgroundColor: THEME.COLORS.TRANSPARENT,
+    position: 'absolute',
+    alignSelf: 'center',
+    top: 17,
+    zIndex: 1
+  }
+}
+
+const styles: typeof rawStyles = StyleSheet.create(rawStyles)
```
Comment (andreyvEze) [inline src/modules/UI/components/Slider/Slider.js:138]: use theme

File: src/modules/UI/components/Slider/Slider.js (hunk 1 above)
Comment (andreyvEze) [reply src/modules/UI/components/Slider/Slider.js:26]: make SLIDER_WIDTH either prop or const

File: src/modules/UI/components/Slider/Slider.js [hunk 4]
```diff
@@ -23,41 +27,55 @@ type Props = {
...
-        runOnJS(onSlidingComplete)()
-      } else {
-        translateX.value = withTiming(UPPER_BOUND)
+      if (!sliderDisabled) {
+        isSliding.value = false
+
+        if (translateX.value < completePoint) {
+          runOnJS(onSlidingComplete)()
+        } else {
+          translateX.value = withTiming(upperBound)
```
Comment (andreyvEze) [inline src/modules/UI/components/Slider/Slider.js:77]: change algo

**Review by thehobbit85 — APPROVED** [review]

**Pull request merged**